### ISSUE TITLE
Improve the file logging to match Trace tab logging

### ISSUE THREAD

**Issue by NiceGuyIT** [Ylianst/MeshCentral#3589]
I enabled logging in my `config.json`.

```json
    "log": [
        "cookie",
        "dispatch",
        "main",
        "peer",
        "web",
        "webrequest",
        "relay",
        "webrelaydata",
        "webrelay",
        "mps",
        "mpscmd",
        "swarm",
        "swarmcmd",
        "agentupdate",
        "agent",
        "cert",
        "db",
        "email",
        "amt",
        "httpheaders",
        "websocket"
    ],
```

When looking at the `log.txt`, I noticed it didn't have near the information the Trace tab has on the website. Here's what's logged in the `log.txt`.

```text
1:12:57 PM - webrequest: (172.30.0.10) /scripts/common-0.0.1.js
1:12:57 PM - httpheaders: 
1:12:57 PM - webrequest: (172.30.0.10) /scripts/u2f-api.js
1:12:57 PM - httpheaders: 
1:12:57 PM - webrequest: (172.30.0.10) /styles/style.css
1:12:57 PM - httpheaders: 
1:12:57 PM - webrequest: (172.30.0.10) /scripts/common-0.0.1.js
1:12:57 PM - httpheaders: 
1:12:57 PM - webrequest: (172.30.0.10) /scripts/u2f-api.js
1:12:57 PM - httpheaders: 
1:12:57 PM - webrequest: (172.30.0.10) /welcome.png
1:12:57 PM - httpheaders: 
1:12:57 PM - webrequest: (172.30.0.10) /welcome.png
1:12:58 PM - dispatch: DispatchEvent
1:12:58 PM - dispatch: DispatchEvent
```

Here's a few lines from the Trace tab.
```text
1:15:57 PM - HTTPHEADERS: GET, /scripts/ol3-contextmenu.js, {"host":"mesh.example.com","connection":"upgrade","x-forwarded-host":"mesh.example.com:443","x-forwarded-for":"172.30.0.10, 172.30.0.119","x-forwarded-proto":"https","user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:96.0) Gecko/20100101 Firefox/96.0","accept":"*/*","accept-encoding":"gzip, deflate, br","accept-language":"en-US,en;q=0.5","cache-control":"max-age=0","cookie":"xid=eyJ1c2VyaWQiOiJ1c2VyLy9maXlrdXF6bCIsImlwIjoiMTcyLjMwLjAuMTAiLCJ4IjoicGUyNGlwMlAiLCJ0IjoyNzM5NzA5NX0=; xid.sig=RUYzB_XCZoiZtU5TnUtZ4jXpdfo","if-modified-since":"Mon, 24 Jan 2022 13:04:36 GMT","if-none-match":"W/\"2c8c-17e8c30346a\"","sec-fetch-dest":"script","sec-fetch-mode":"no-cors","sec-fetch-site":"same-origin","x-forwarded-port":"443","x-forwarded-server":"ns-v19-proxy","x-real-ip":"172.30.0.10"}
1:15:57 PM - WEBREQUEST: (172.30.0.10) /scripts/ol.js
1:15:57 PM - HTTPHEADERS: GET, /scripts/ol.js, {"host":"mesh.example.com","connection":"upgrade","x-forwarded-host":"mesh.example.com:443","x-forwarded-for":"172.30.0.10, 172.30.0.119","x-forwarded-proto":"https","user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:96.0) Gecko/20100101 Firefox/96.0","accept":"*/*","accept-encoding":"gzip, deflate, br","accept-language":"en-US,en;q=0.5","cache-control":"max-age=0","cookie":"xid=eyJ1c2VyaWQiOiJ1c2VyLy9maXlrdXF6bCIsImlwIjoiMTcyLjMwLjAuMTAiLCJ4IjoicGUyNGlwMlAiLCJ0IjoyNzM5NzA5NX0=; xid.sig=RUYzB_XCZoiZtU5TnUtZ4jXpdfo","if-modified-since":"Mon, 24 Jan 2022 13:04:36 GMT","if-none-match":"W/\"7dd9d-17e8c30346a\"","sec-fetch-dest":"script","sec-fetch-mode":"no-cors","sec-fetch-site":"same-origin","x-forwarded-port":"443","x-forwarded-server":"ns-v19-proxy","x-real-ip":"172.30.0.10"}
1:15:57 PM - WEBREQUEST: (172.30.0.10) /scripts/charts.js
1:15:57 PM - HTTPHEADERS: GET, /scripts/charts.js, {"host":"mesh.example.com","connection":"upgrade","x-forwarded-host":"mesh.example.com:443","x-forwarded-for":"172.30.0.10, 172.30.0.119","x-forwarded-proto":"https","user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:96.0) Gecko/20100101 Firefox/96.0","accept":"*/*","accept-encoding":"gzip, deflate, br","accept-language":"en-US,en;q=0.5","cache-control":"max-age=0","cookie":"xid=eyJ1c2VyaWQiOiJ1c2VyLy9maXlrdXF6bCIsImlwIjoiMTcyLjMwLjAuMTAiLCJ4IjoicGUyNGlwMlAiLCJ0IjoyNzM5NzA5NX0=; xid.sig=RUYzB_XCZoiZtU5TnUtZ4jXpdfo","if-modified-since":"Mon, 24 Jan 2022 13:04:36 GMT","if-none-match":"W/\"ce805-17e8c3033b6\"","sec-fetch-dest":"script","sec-fetch-mode":"no-cors","sec-fetch-site":"same-origin","x-forwarded-port":"443","x-forwarded-server":"ns-v19-proxy","x-real-ip":"172.30.0.10"}
1:15:57 PM - WEBREQUEST: (172.30.0.10) /scripts/u2f-api.js
```

The problem is stemming from `Array.prototype.slice.call` [meshcentral.js](https://github.com/Ylianst/MeshCentral/blob/master/meshcentral.js#L3131). Is it possible to improve the logging so that `log.txt` is closer to what is logged in the Trace tab?

MeshCentral v0.9.73

**Comment by Ylianst** [comment]
Oh, most certainly a bug! Thanks for reporting this. Hopefully I will fix this tomorrow.

**Comment by Ylianst** [comment]
Just fixed it, will be in MeshCentral v0.9.78.

**Comment by JSuenram** [comment]
@NiceGuyIT 
I can confirm this is fixed. Pls close....
